mParticle-Apptentive.xcodeproj/project.pbxproj

```diff
@@ -3,17 +3,32 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 46;
+	objectVersion = 52;
 	objects = {
 
 /* Begin PBXBuildFile section */
-		DB695FB31F5F549A00A7F4CF /* Apptentive.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = DB695FB21F5F549A00A7F4CF /* Apptentive.framework */; };
+		D37EFA8724F430580091B75B /* Apptentive.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = DB695FB21F5F549A00A7F4CF /* Apptentive.framework */; platformFilter = ios; };
+		D37EFA8824F430580091B75B /* Apptentive.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = DB695FB21F5F549A00A7F4CF /* Apptentive.framework */; platformFilter = ios; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
```

Comment:
Unless there's some special reason it's necessary for Apptentive, I think we can avoid adding 'Embed Frameworks'... I noticed e.g. AppsFlyer doesn't have it in project settings.